jfx_bridge/bridge.py

```diff
@@ -110,6 +110,12 @@ class ThreadingTCPServer(socketserver.ThreadingMixIn, socketserver.TCPServer):
 GLOBAL_BRIDGE_SHUTDOWN = False
 
 
+_EXTERNAL_OVERRIDES = {}
+
+def register_overrides(overrides):
```

Comment:
So I generally like this API. It needs a little more documentation, but I think it makes sense.

However, I think it'd work a lot better as part of the BridgeConn object, and also exposed in the BridgeClient object (the same way remote_eval is exposed by BridgeClient). That means that you'd be able to register_overrides for a specific bridge instance - I've got a couple of use cases where I'll have multiple bridges of different types running at the same time, and I don't think you'd always want the same overrides to affect all the bridges.